### DISCUSSION TITLE
QHY 294c Pro 11M and 47M settings - I know I am thick !

### DISCUSSION THREAD

**Discussion by GitHubbet** [Stellarium/stellarium#2538] [Q&A]
Hi
I have this camera and I would like to know how to set it for 11M and 47M - i.e. this camera have 2 hardware settings as chip pixel size is 4.63 (depending on manufacturers specs) at 11M, but since it is a Quad chip, you can set it to multiple 4x pixels and becomes a 2.3µ.

I cannot seem to be able to set it up.

Before, at least I used the Pixel Size that has now been taken off.

or alternatively, how do I install older version of Oculars?

Thank you
M

PS I keep going on-off with astro imaging and I keep forgetting things! I am a constant newbie!

**Comment by gzotti** [comment]
You can configure the 47MPix camera based on chip size and pixel count and use 2x2 binning configuration for 1MPix. Alternatively, configure two cameras.

**Reply by GitHubbet** [reply]
Thanks for reply.
Pixel size is not any more in the last version, that is why I am asking. I normally set/repeat cameras at different settings.

I just do not know how to do that without the Pixel Size in Oculars.

**Comment by gzotti** [comment]
The pixel size field never worked correctly. The relevant entries are chip size and pixel count.
Simply divide chip width by pixel count to get pixel size (to check data sheet). Or multiply pixel width by pixel count to get chip size.

**Reply by GitHubbet** [reply]
OK, thanks.
I think only thing I can do is :
at 11M set standard sizes
at 47M set chip (in mm) twice the size and it reduce image 'view' by x4 and I switch to correct Telescope for best CCD imaging pixel size suitability.
It seems to be OK when I use 12dStrings site, by simply changing pixel size...
Never mind - thanks

🧠🔥🧯

**Reply by gzotti** [reply]
What is the problem?

A quick search provides https://www.baader-planetarium.com/de/qhy-294m-c-pro-medium-size-cooled-cmos-kamera.html indicating a 

* CHIP SIZE of 19,28 x 12,95 millimeters
* 4.164 x 2.796 Pixels (11.7MPix with 4.63µm pixel size) at 14 bit depth. 
* 46,8 MPix at 2.315 µm pixel size at 12 bit depth. 

They do no list the pixel count explicitly.  Given that 2.315µm is half of 4.63µm, we can trivially conclude that the pixel count at 46.8MPix is double that of the 11.7MPix resolution, in both dimensions.  THE CHIP SIZE DOES NOT CHANGE.

**Comment by GitHubbet** [comment]
Sorry to have wasted your time.
It is not about calculations, I have no problems in doing these simple calculations, I am good at maths - it is the concept I got wrong, I guess!.

I was only setting Ocular up again after I uninstalled (and cleaned old data) and re-installed newest version of Stellarium 64bit and I was checking the 'views' using QHY 294c - both modes on two different telescopes:

AR152 988mm - at 11M due to longer FL
72ED 420mm - at 47M

Then I thought: how do I see same Object with same telescope, but at 11M and 47M ?
Just to see the framed views.

There I got confused.

As mentioned, before I used the pixel size - et voila' was working OK.

That was all

Thanks
